app/controllers/measures/additional_codes_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Measures
```

Comment:
We will eventually need to build controllers to edit all this reference data, I wonder if we should not nest these within measure, we can refactor that later though.